tokio-trace/src/span.rs

```diff
@@ -14,7 +14,7 @@
 //! # #[macro_use] extern crate tokio_trace;
 //! # fn main() {
 //! let my_var: u64 = 5;
-//! let mut my_span = span!("my_span", my_var = &my_var);
+//! let mut my_span = span!(tokio_trace::Level::TRACE, "my_span", my_var = &my_var);
```

Comment:
Nit: can we have the examples `use tokio_trace::Level` and then refer to the level as `Level::TRACE`, instead of using the fully qualified path?